usersession/agent/rest_api_test.go

```diff
@@ -54,6 +57,7 @@ var _ = Suite(&restSuite{})
 func (s *restSuite) SetUpTest(c *C) {
 	s.BaseTest.SetUpTest(c)
 	s.DBusTest.SetUpTest(c)
+	dirs.SnapDesktopFilesDir = c.MkDir()
```

Comment:
this needs to be removed to avoid inconsistency, `dirs.SetRootDir` below adjusts all dirs for testing

Reply:
Done.